gstreamer-pbutils/src/auto/discoverer_info.rs

```diff
@@ -0,0 +1,186 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir @ 47eb915)
+// from gir-files (https://github.com/gtk-rs/gir-files @ ???)
+// DO NOT EDIT
+
+use DiscovererResult;
+use DiscovererSerializeFlags;
+use DiscovererStreamInfo;
+use ffi;
+use glib;
+use glib::object::IsA;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use gst;
+use std::mem;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct DiscovererInfo(Object<ffi::GstDiscovererInfo>);
+
+    match fn {
+        get_type => || ffi::gst_discoverer_info_get_type(),
+    }
+}
+
+impl DiscovererInfo {
+    pub fn from_variant(variant: &glib::Variant) -> Option<DiscovererInfo> {
+        assert_initialized_main_thread!();
+        unsafe {
+            from_glib_full(ffi::gst_discoverer_info_from_variant(variant.to_glib_none().0))
+        }
+    }
+}
+
+unsafe impl Send for DiscovererInfo {}
+unsafe impl Sync for DiscovererInfo {}
+
+pub trait DiscovererInfoExt {
+    fn copy(&self) -> DiscovererInfo;
+
+    fn get_audio_streams(&self) -> Vec<DiscovererStreamInfo>;
+
+    fn get_container_streams(&self) -> Vec<DiscovererStreamInfo>;
+
+    fn get_duration(&self) -> gst::ClockTime;
+
+    #[cfg(any(feature = "v1_14", feature = "dox"))]
+    fn get_live(&self) -> bool;
```

Comment:
1.14 feature here, let's stay at 1.12 API-wise for now (but annotation fixes from 1.14 are fine).